test/postControllerTest.js

```diff
@@ -44,10 +44,11 @@ describe('postController', () => {
           sinon.assert.calledWith(postService.getAllPosts);
         })
     ));
-    it('should return resolved result of "postService.getAllPosts" to view', () => (
+    it('should return resolved result of "postService.getAllPosts" as json object', () => (
       request(app)
         .get('/')
         .expect(200)
+        .expect('Content-Type', /json/)
         .expect(/post1/)
```

Comment:
Now that the exact structure of what you are returning is known, these vague asserts can be improved. Maybe something like [assert.deepStrictEqual](https://nodejs.org/api/assert.html#assert_assert_deepstrictequal_actual_expected_message) will work

Reply:
Would using ```.expect``` to expect the json object be acceptable as well. ```deepStrictEqual``` compares all fields, and because I am using the mongo model, an random string is generated for the ```"_id"``` field.